### PR TITLE
don't save previous screen positions for wheel events

### PR DESCRIPTION
**Pull request by hpohl**
See https://github.com/kripken/emscripten/issues/4996

### CODE REVIEW

**Comment by kripken** [comment]
Is this a workaround for a bug in a specific browser? The [mdn docs](https://developer.mozilla.org/en-US/docs/Web/Events/wheel) suggest that `screenX/Y` should be valid for a wheel event too.

**Comment by hpohl** [comment]
Wheel events have incorrect `screenX/screenY` values when the pointer is locked on Chrome for Linux/Mac. I guess it is a bug, yes. A minimal test case might be useful.

**Comment by kripken** [comment]
Ok. I'm fine with merging a workaround, but let's first file a bug for chrome, and add a link to that bug in the code with a comment, so we know we can track whether we can remove it later.

**Comment by hpohl** [comment]
Absolutely: https://bugs.chromium.org/p/chromium/issues/detail?id=699956

**Comment by juj** [comment]
Can you also gate it out from 'mousewheel' event as well? See https://github.com/kripken/emscripten/blob/master/src/library_html5.js#L1123. Also, please put a comment in the code file, linking to this PR where this discussion occurs with the link to upstream. We really don't expect to get wheel events with different mouse coordinates, so this sounds like an ok change to have permanently.

**Comment by hpohl** [comment]
@juj done

**Comment by juj** [comment]
Thanks, looks great! Could you add yourself also to https://github.com/kripken/emscripten/blob/incoming/AUTHORS (if not already)?

**Comment by hpohl** [comment]
@juj done :)

File: src/library_html5.js [hunk 1]
```diff
@@ -262,8 +262,12 @@ var LibraryJSEvents = {
         {{{ makeSetValue('eventStruct', C_STRUCTS.EmscriptenMouseEvent.targetX, '0', 'i32') }}};
         {{{ makeSetValue('eventStruct', C_STRUCTS.EmscriptenMouseEvent.targetY, '0', 'i32') }}};
       }
-      JSEvents.previousScreenX = e.screenX;
-      JSEvents.previousScreenY = e.screenY;
+      // wheel and mousewheel events contain wrong screenX/screenY on chrome/opera
+      // https://github.com/kripken/emscripten/pull/4997
```
Comment (kripken) [inline src/library_html5.js]: shouldn't this be a link to the chromium bug?

Comment (hpohl) [inline src/library_html5.js]: @juj sounded like I should like to this PR which contains the link to the bug report. But I can change that, if you want.

Comment (kripken) [inline src/library_html5.js]: Hmm, @juj, did you mean to link to this PR, or to the upstream issue? We usually link straight to upstream (finding the PR can be done from the commit/blame/etc.)?

**Comment by kripken** [comment]
I merged this now and added a second link, so we have  both.

**Pull request merged**